### PR TITLE
feature/day03 : move day03 to day04 and update day03

### PR DESCRIPTION
**Pull request by 42ghan**
day03 & day04 끝

### CODE REVIEW

File: pages/day03.html [hunk 1]
```diff
@@ -229,16 +229,20 @@ <h3 class="content__subhead xlarge text-bold">Basic Instruction</h3>
...
             <h3 class="content__subhead xlarge text-bold">Foreword</h3>
             <p class="content__para">
-              대박사건 대박사건! 오늘은 단 한문제로 끝납니다! 그리고 놀라운
-              CSS의 기능에 감탄하게 될 겁니다.
+              여러분... 아이유 콘서트 가 보셨나요? 세상에 이렇게 좋은 콘서트가
+              있을 수 없답니다. 하핫! 저 작은 체구에서 노래를 5시간을 연달아
+              부른다니까요? 진짜 경이로운데 노래가 아주 그냥 하나하나 주옥같아서
+              행복으로 가득한 하루를 보낼 수 있답니다. 이 글을 쓰고 있는 저도
+              특별한 생각 없이 갔던 콘서트가 준 몽실몽실한 기억 덕에 아이유 팬이
+              되었답니다 :)
```
Comment (yongjulejule) [inline pages/day03.html:237]: 진짜 필참.

**Review by bokyungkim — APPROVED** [review]
멋쟁이 지한~~~~

**Comment by yongjulejule** [comment]
와~~~~~~~~~~~~~~~~ 굿굿굿

**Review by yongjulejule — APPROVED** [review]
approve:)

**Pull request merged**